code/modules/admin/admin.dm

```diff
@@ -862,4 +862,34 @@ proc/kick_clients_in_lobby(var/message, var/kick_only_afk = 0)
 				C << message
 			kicked_client_names.Add("[C.ckey]")
 			del(C)
-	return kicked_client_names
\ No newline at end of file
+	return kicked_client_names
+
+
+/datum/admins/proc/cmd_ghost_drag(var/mob/dead/observer/frommob, var/mob/living/tomob)
+
+	//this is the exact two check rights checks required to edit a ckey with vv.
+	if (!check_rights(R_VAREDIT,0) || !check_rights(R_SPAWN|R_DEBUG,0))
+		return 0
+
+	if (!frommob.ckey)
+		return 0
+
+	var/question = ""
+	if (tomob.ckey)
+		question = "This mob already has a user ([tomob.key]) in control of it! "
+	question += "Are you sure you want to place [frommob.name]([frommob.key]) in control of [tomob.name]?"
+
+	var/ask = alert(question, "Place ghost in control of mob?", "Yes", "No")
+	if (ask != "Yes")
+		return 1
+
+	tomob.ghostize(0)
+
+	message_admins("<span class='adminnotice'>[key_name_admin(usr)] has put [frommob.ckey] in control of [tomob.name].</span>")
+	log_admin("[key_name(usr)] stuffed [frommob.ckey] into [tomob.name].")
+	feedback_add_details("admin_verb","CGD")
+
+	tomob.ckey = frommob.ckey

```

Comment:
Edit, ya, i don't want transfer_to, i did some testing.

Without transfer_to:

If both mobs have no mind, a new one gets made. -expected behavior

If both mobs have a mind, the to mob's mob is now used. -expected behavior

if the from mob has a mind, but the to mob does not, a new one gets made -expected behavior in most cases (spawning special rules for events)

if the from mob has no mind, but the tomob does, the tomob's mind gets used -expected behavior.

With transfer_to:

If both mobs have no mind, a new one gets made. -expected behavior

If both mobs have a mind, the from mob's mob is now used. -not expected behavior in most cases (spawning special roles for events, putting ghosts in afk user's bodies for critical antags)

if the from mob has a mind, but the tomob does not, a new one gets made -not expected behavior in most cases (spawning special roles for events)

if the from mob has no mind, but the tomob does, a new mind would have to get made, and used. -not expected behavior in most cases. (putting ghosts in afk user's bodies for critical antags).